net/host_test.go

```diff
@@ -59,9 +59,13 @@ func (s *mockSwapState) Exit() error {
 }
 
 func newHost(t *testing.T, port uint16) *host {
+	dir, err := os.MkdirTemp(os.TempDir(), "atomicswap*")
+	require.NoError(t, err)
```

Comment:
I'd replace these 2 lines with:
```
	dir := t.TempDir()
```
You'll get a unique directory that includes the test's name, and the directory will be cleaned up on success.

Reply:
nice, updated!